doc/src/input/md.md

```diff
@@ -184,3 +184,5 @@ controls = [
 
 - The `RemoveTranslation` control removes the global system rotation;
 - The `RemoveRotation` control removes the global system translation.
+- The `Rewrap` control rewrap all molecules' centers of mass to lie within the
```

Comment:
rewrap**s**

Reply:
-_- ... do you know a good grammar checker for english? I really need one installed.

Reply:
Mh no not really...although I get constantly spammed with ads from [grammarly](https://www.grammarly.com) but I have no idea how/if it works with github :)